packages/slice-machine/src/modules/simulator/index.ts

```diff
@@ -206,96 +146,63 @@ function* checkSetupSaga(
       checkSimulatorSetup
     );
 
-    // All the backend checks are ok ask for the frontend Iframe check
-    if ("ok" === setupStatus.manifest && "ok" === setupStatus.dependencies) {
+    if (setupStatus.manifest === "ok") {
```

Comment:
Wouldn't it be easier to use a boolean here instead of the string? As far as I can tell "ko" and null and treated the same so could be condensed into `isManifestOk`

Reply:
I'm a bit unclear with this and I remember I raised the same issue when I reviewed this part of the code. But differentiating between ko and null makes things easier when you create a race condition between different events (iframeCheckOk, iframeCheckKO and timeout). It's treated the same way in the saga but it offers more flexibility in the UI. Still not super clean indeed